\documentclass[../main.tex]{subfiles}
\begin{document}

\chapter{Introduction}
\labch{intro}

Here I should describe the state of the art on the subject.

\section{The old nature-nurture debate}

%Environment can influence gene expression and epigenome as well. Also 
%the genome, for instance UV rays cause mutations. Most of the time the 
%environmental effects are random, but not always (\eg UV rays, 
%smoking...).

\section{Genome-wide association studies and eQTL (classics)}

\subsection{Pros (successes)}

\subsection{Cons (limits)}

\section{Fine mapping, COLOC?, LDSC? (modern state of the art)}

%----

%https://www.ebi.ac.uk/gwas/home

%https://www.broadinstitute.org/news/after-decade-genome-wide-association-st

%"finding the missing heritability"

%----

%Genes can do few things: either they bind proteins whith a structural 
%or regulatory function (and when it is structural, it can be 
%regulating: TAD are coregulated), or they are transcribed, starting a 
%series of biochemical reactions that ultimately lead to functional 
%molecules, be they RNA or proteins. The complexity stems from the 
%interactions of many genes together and with the environment

%----

%https://www.nature.com/articles/ng0508-489

%Combination of alleles may have specific effects both if they occur at 
%the same locus (dominance) or at different loci (epistasis). 

%There is a nice picture showing the statistical power of association 
%studies as a function of effect size and population size.

%One could find genes near markers associated to a phenotype and look 
%for overrepresented pathways.

%\enquote{The main conclusion emerging from the current studies is that 
%GWAS are able to robustly identify common variants that are associated 
%with height but that the effect sizes of individual variants are small, 
%so that very large sample sizes are needed to detect associations 
%reliably. Single laboratories are unlikely to have sufficient sample 
%sizes to do powerful studies on their own, and the trend in human 
%complex trait mapping has been to create consortia of research groups 
%and even consortia of consortia.}

%At the same time, among the full sequences now available there are so 
%many variants that trying to associate them with anything is very 
%difficult. Statistics is not enough in this case.

%----

%Height and most other quantitative traits are influenced by many 
%variants of small effects.

%gamazon2015:Gwas have found many associations, but a large sample size 
%is needed.

%other limit of gwas: they study single variants, but sometimes the 
%disease manifest only when there is a certain \textit{combination} of 
%variants.

%gamazon2015: Gwas on their own are not enough (cite 
%https://www.nature.com/articles/nature08494). in particular, there is a 
%missing link between the variant and the disease: how (not why, 
%\textit{how}) does the variant make one individual more susceptible to 
%a disease? It is not true that the nearest gene is always involved.

%fine mapping? (it may be necessary also for TWAS.)

%gamazon2015: Many SNPs are found in regulatory regions, as evinced by 
%the fact that they overlap with DNaseI sites (is this true? read Gusev, 
%A. et al. Regulatory variants explain much more heritability than 
%coding variants across 11 common diseases. bioRxiv 004309 (21 April 
%2014).), and that they often are found in eQTL (see Nicolae, D.L. et 
%     al. 
%Trait-associated SNPs are more likely to be eQTLs: annotation to 
%enhance discovery from GWAS. PLoS Genet. 6, e1000888 (2010).)

%gamazon2015: eQTL mapping has shown that intermediate phenotypes, 
%especially gene expression, are important. New projects are generating 
%huge amounts of epxression data: ENCODE, GTEx, GEUVADIS, to name a few.

%gamazon2015: they provide a way to aggregate SNPs in a biologically 
%meaningful way, since they combine those variants that influence the 
%expression of a gene. In general, either the phenotype is influenced by 
%a small number of variants (one in the extreme case of mendelian 
%diseases) with a large effect, or by a large number of variants of 
%small effect. PrediXscan should districate this entanglement.

%http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000888 
%says that gwas are also eqtl. it is the basis of everything.

%gamazon2015: multiple testing problems reduced.

\end{document}
