\documentclass[../main.tex]{subfiles}
\begin{document}

\chapter{Linkage Disequilibrium}
\labch{ld}

\section{Definition}

Linkage disequilibrium is the non-random association of two alleles. In
other words, two alleles can be found together in more (or less)
individuals than what is expected by chance.

An important concept is that linkage disequilibrium always refers to
alleles in the same haplotype, i.e. in the alleles which are inherited
together from a single parent. For haployd organisms, therefore,
we will consider the genome of the individual as haplotype, but for
diploid individuals, such as humans, we can consider the gametes.

\section{http://www.els.net/WileyCDA/ElsArticle/refId-a0005427.html}
\labsec{ld_balance}

Though most of the alleles in linkage disequilibrium are found on the 
same chromosome, this is not always the case, for linkage disequilibrium 
can arise from many different processes: genetic mutations, selection, 
genetic drift and gene flow, to name a few. \textit{De novo} genetic 
mutation, where a new allele is created is an important one in linkage 
disequilibrium. It is clear that, among the descendants of the 
individual where the mutation has appeared, this allele is in linkage 
disequilibrium with its neighbours, for it always occurs with the same 
alleles.

Now let us consider selection; it is not difficult to imagine that the 
function of a protein complex made by two subunits is affected by which 
allele is present for each of the two genes involved in the complex, 
therefore it is conceivable that those combinations that work best will 
be selected. Linkage disequilibrium can be experimented by non-coding 
regions as well, but such situation is more complex to depict.

Another interesting process which influence the equilibrium between loci 
is recombination, which acts during meiosis. Recombination is different 
from the previous processes, for it does not create disequilibrium, but 
rather dissipates it, bringing the alleles back to equilibrium with a 
rate which is a function of the phyiscal distance between the loci. It 
follows that if we know the recombination rate between two loci and 
their linkage disequilibrium, we can estimate the time at which the 
alleles appeared. Another consequence of recombination is the creation 
of recombinant phenotypes, which occur indeed when disequilibrium is 
broken. Let us suppose that allele \geno{A} is in perfect linkage 
disequilibrium with allele \geno{B}, so that they are always found 
together in the same haplotype; the same holds for \geno{a} and 
\geno{b}. When two different haplotypes, one having \geno{AB} and the 
other \geno{ab}, are found in the same diploid individual, a 
recombination event can occur between these two sister chromatides such 
as to cut the DNA right between locus A and locus B. In this case, a new 
phenotype, deemed "recombinant", would be created.

\todo{Relation of LD and allele frequency? If one locus has only one 
allele (limit case of monomorphic locus), LD will always be perfect. On 
the other hand, what happens when the frequency of an allele is low?}

To sum up, disequilibrium between alleles is the product of many 
balancing processes; recombination, on the other hand, balances those 
processes so as to bring equilibrium. A corollary of these observations 
is that if two alleles are in disequilibrium, in all probability there 
is a reason behind, be it that one of the allele is newly appeared or 
that there is selection to maintain the two alleles together.

Not only does linkage disequilibrium suggest a relationship between 
alleles, it is also the basis for association studies. These are 
statistical analysis whose purpose is to find out alleles occurring more 
often than expected by chance in individuals bearing a certain 
phenotype, implying that such alleles are somewhat correlated with the 
phenotype. Traditionally, the phenotypes investigated in association 
studies are pathological ones, both because they are more interesting 
and because they are easier to study, provided that they are not complex 
diseases. \todo{This paragraph is weak.}

Linkage can be calculated for more than two alleles, but it is rarely 
done. I do not know whether it is due to the complexity of the 
calculation or because it has a poor biological relevance, but I would 
bet that it would be an interesting area to explore.

\section{http://www.handsongenetics.com/PIFFLE/LinkageDisequilibrium.pdf}

Let us consider a simplified model where all loci are independent, 
taking two loci, A and B, both of which are biallelic, carrying alleles 
\geno{A,a} and \geno{B,b}, respectively; also, in our model the 
population is a set of diploid individuals, each of which can be 
described by a pair of haplotypes. In all the gametes of such 
population, allele \geno{A} has a frequency of \geno{p_A}, and allele 
\geno{B} of \geno{p_B}, so it follows from simple probabilistic 
considerations that in a diploid organism the probability of finding 
both alleles is \geno{p_AB = p_a p_b}. This result, however, holds true 
only as long as the loci are independent.

In a real population, \geno{p_AB} may well differ from the product of 
the allele frequencies, so it has been introduced the `linkage 
coefficient', which is defined as

\begin{equation}
\label{eq:linkagecoeff}
D = p_AB - p_A p_B\,,
\end{equation}

as a measure of this difference.

Let us try to explain how \(D\) can be differen from \(0\) by 
considering genetic mutations, which are one of the processes through 
which linkage disequilibrium can arise, as we said in 
\refsec{ld_balance}. At the beginning, loci A and B are monomorphic, 
that is they both have only one allele. You can imagine each allele as 
being a sequence of letters, or a word, in which case each locus only 
has one word; for the sake of our metaphor, let us say that locus A 
holds allele (word) ELEPPANT while locus B holds allele (word) CANGAROO. 
Each and every individual has two haplotypes, and each haplotype holds 
the word ELEPHANT and CANGAROO. At one point, one individual exposes 
himself to the open sun, and an UV ray causes a \textit{mutation} in his 
DNA, so that now, in one of his haplotypes, locus A reads ELEPHANT, 
whereas in the other haplotype it is still ELEPPANT. Since the mutation 
happened in his gametes, his offspring inherits it. It turns out that 
having this mutation makes the individuals fitter, so its frequency 
increases overtime, until it becomes \geno{p_A} (remember: allele 
frequency is the number of gametes having that allele devided by the 
total number of gametes in the population). The frequency of the 
ELEPPANT allele, instead, is \geno{p_a}. At another point, another 
mutation happens in a haplotype with allele \geno{a}, creating the 
allele CAGGAROO at locus B. Let us call allele CANGAROO \geno{B}, with 
frequency \geno{p_B}, and the other \geno{b}. Now, then, there are two 
alleles for each locus, and three possible haplotypes: \geno{AB}, 
\geno{ab}, and \geno{aB}, the latter of which is the original one. These 
haplotypes can originate six possible genotypes, namely the combination 
with repetition of the three alleles for the two haplotypes of each 
individual. Let us consider haplotype \geno{ab}. In the generation in 
which the mutation CAGGAROO appears, there are 5 individuals, \ie 10 
gametes, and the allele frequencies are showed in the table at the 
margin.

\begin{margintable}
\begin{tabular}{|c|c|}
	\hline
	Allele	& frequency	\\
	\hline
	A		& 0.7		\\
	a		& 0.3		\\
	B		& 0.9		\\
	b		& 0.1		\\
	\hline
\end{tabular}
\caption{Sample alleles and their frequency}
\end{margintable}

There is only one haplotype \geno{ab}, for that mutation has just 
appeared, so that \geno{p_ab = 0.1}. However, \geno{p_a p_b = 0.3 \cdot 
0.1 = 0.03}, which is much smaller than \geno{p_ab}, therefore, the 
  linkage coefficient is \(D = 0.1 - 0.03 = 0.07\). Selection and other 
  processes will act to increase or decrease such disequilibrium.

Now let us consider the other process, recombination, which as we said 
always decreases disequilibrium.

\end{document}
